support-frontend/conf/PROD.public.conf

```diff
@@ -16,7 +16,7 @@ paymentApi.url="https://payment.guardianapis.com"
+play.filters.headers.contentSecurityPolicy = "default-src 'self' https://cdn.privacy-mgmt.com https://gdpr-tcfv2.sp-prod.net https://ccpa-service.sp-prod.net https://ccpa-notice.sp-prod.net https://consent.theguardian.com https://ccpa.sp-prod.net https://services.postcodeanywhere.co.uk  https://contribution-reminders.support.guardianapis.com/ https://stripe-intent.support.guardianapis.com members-data-api.theguardian.com metric-push-api-prod.support.guardianapis.com www.paypalobjects.com www.paypal.com t.paypal.com www.sandbox.paypal.com js.stripe.com https://payments.amazon.com https://payments-sandbox.amazon.com https://coin.amazonpay.com https://api-cdn.amazon.com static-na.payments-amazon.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk uploads.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com polyfill.io www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube-nocookie.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io www.dwin1.com data: wss: 'unsafe-inline' q.stripe.com payment.guardianapis.com https://interactive.guim.co.uk/ https://www.theguardian.com/ https://theguardian.com/ https://contribution-referrals.support.guardianapis.com/referral"
```

Comment:
All scripts executed by Sourcepoint should be through `sourcepoint.theguardian.com`.

```
+ https://sourcepoint.theguardian.com
- https://cdn.privacy-mgmt.com https://gdpr-tcfv2.sp-prod.net https://ccpa-service.sp-prod.net https://ccpa-notice.sp-prod.net https://consent.theguardian.com https://ccpa.sp-prod.net
```

Reply:
@mxdvl thanks for feedback, I had to add these paths to the CSP to make it work. Regarding the rest of the policy, this is the policy in place right now (aside from the additional paths) and I'm not making significant changes to it here in this PR.